HelpSource/Guides/Internal-Snooping.schelp

```diff
@@ -5,27 +5,36 @@ categories:: Debugging
 You can inspect much of the internal structure of the class library and other data structures.
 This can often be useful for research and debugging purposes.
 
-warning:: The keyboard shortcuts mentioned here only applies to the Mac SC.app. This document should probably be updated. ::
+note:: The keyboard shortcuts mentioned here only applies to the standard supecollider IDE.::
 
 section:: Class Definitions, Implementations, and References
 
-Selecting the name of any Class (e.g. Object) and typing cmd-j will open its class definition file.
+subsection::cmd-i
+Selecting the name of any Class (e.g. Object) and typing strong::cmd-i:: will open its class definition file.
 
-note:: Any keypresses that open files identify the files using SuperCollider strings, which may contain only ASCII characters up to code point 127. If any part of the path to your SuperCollider installation contains character codes greater than 127 (such as the old Mac convention of identifying program folders with option-F), automatic file opening will fail. ::
 
-Selecting the name of any method (e.g. play) and typing cmd-y will open a window showing all implementations of that method and their arguments. Selecting one of those classes and methods (e.g. Sample:play) and typing cmd-j will open the class definition at that method. (Note that cmd-y only shows implementations, and does not indicate inheritance).
+subsection::cmd-u
+Selecting the name of any method (e.g. play) and typing strong::cmd-u:: will open a window showing all implementations of that method and their arguments. Selecting one of those classes and methods (e.g. Sample:play) and typing cmd-j will open the class definition at that method. (Note that cmd-y only shows implementations, and does not indicate inheritance).
+
+subsection::cmd-shift-u
+Selecting any text (e.g. Window or asStream) and typing strong::cmd-shift-u:: will open a window showing all references to the selected text, i.e. each place it is used within the class library. (This will not find methods calls compiled with special byte codes like 'value'.)
 
-Selecting any text (e.g. Window or asStream) and typing shift-cmd-y will open a window showing all references to the selected text, i.e. each place it is used within the class library. (This will not find methods calls compiled with special byte codes like 'value'.)
+note:: Any keypresses that open files identify the files using SuperCollider strings, which may contain only ASCII characters up to code point 127. If any part of the path to your SuperCollider installation contains character codes greater than 127 (such as the old Mac convention of identifying program folders with option-F), automatic file opening will fail. ::
```

Comment:
Both statements here are inaccurate; SC strings are raw 8-bit (0x0-0xff) with no assumed encoding, and usually treated as UTF-8 particularly by the IDE. I (on macOS) can use cmd-I to open a file named `ƒ.sc`. #2861 will make this warning also obsolete on Windows.